config/actions/generateIndexes.js

```diff
@@ -0,0 +1,99 @@
+const fs = require('fs');
+const findUp = require('find-up');
+const path = require('path');
+const mainFilePattern = 'index.';
+
+// This might be better as a map over dictionary of tagged template fns
+// given that switches evaluate linearly while dictionaries are logarithmic.
+// But this seems trivial, and a switch is a concise readable abstraction
+// for the amount of formats we have at the moment.
+
+function createReferences (fileExt, importPaths) {
+  switch (fileExt) {
+    case 'js':
+      return importPaths.map(p => `export * from './${p}';`).join('\n');
+    case 'scss':
```

Comment:
This is empty, so I thought is was a bug. But of course, it's not (and that's why you don't use switches!). Perhaps add a comment?
```suggestion
    case 'scss': // intentionally empty so that it falls through to less.
```

Reply:
I'm not sure that's a valid argument against switch statements. 😄 
It's the closest thing to a pattern matching feature we have in javascript, other than Map.
This behaviour is also clearly [documented](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#Methods_for_multi-criteria_case) 

That said, we could use a more explicit assertion here, given that we'll be converting this into a dictionary once more formats are added. 

```suggestion
  if (fileExt === 'js') {
    return importPaths.map(p => `export * from './${p}';`).join('\n');
  }
  if (['scss', 'less'].includes(fileExt)) {
    return importPaths.map(p => `@import './${p}';`).join('\n');
  } 
  return undefined;
```